connectors/es/sink.py

```diff
@@ -913,3 +938,20 @@ async def async_bulk(
             enable_bulk_operations_logging=enable_bulk_operations_logging,
         )
         self._sink_task = asyncio.create_task(self._sink.run())
+        self._sink_task.add_done_callback(functools.partial(self.sink_task_callback))
+
+    def sink_task_callback(self, task):
+        if task.exception():
+            self._logger.error(
+                f"Encountered an error in the sync's Sink: {task.get_name()}: {task.exception()}",
```

Comment:
```suggestion
                f"Encountered an error in the sync's {Sink.__class__.__name__}: {task.get_name()}: {task.exception()}",
```

nit: makes renaming a bit safer